proto/src/determined/task/v1/task.proto

```diff
@@ -49,22 +49,26 @@ enum TaskType {
 // Allocation tracks a specific instance of a Task.
 message Allocation {
   option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
-    json_schema: {
-      required: [ "task_id", "state", "is_ready", "allocation_id" ]
-    }
+    json_schema: { required: [ "task_id", "state", "allocation_id", "slots" ] }
   };
   // Unique ID of task associated with the allocation.
   string task_id = 1;
   // The current state of the allocation.
   State state = 2;
   // Whether the allocation is ready to access.
-  bool is_ready = 3;
+  optional bool is_ready = 3;
```

Comment:
is breaking protos here ok?

@stoksc do we have a rule for minor breakages like this? 

I'm not sure if this causes errors with like out of date CLIs for exmaple? I expect not since it goes through the json conversion

Reply:
yeah i think we are totally fine, since it is JSON compatible